draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3652,6 +3652,12 @@ bytes=500-700,601-999
 <sourcecode type="abnf7230"><iref primary="true" item="Grammar" subitem="Content-Location"/>
   <x:ref>Content-Location</x:ref> = <x:ref>absolute-URI</x:ref> / <x:ref>partial-URI</x:ref>
 </sourcecode>
+<t>
+   The field value consists of a single URI-reference. When it has the form
```

Comment:
I don't think this is correct. Content-Location is not defined as a URI-reference because we don't allow it to contain a fragment. So, I think his needs to say "absolute-URI or partial-URI" or "URI reference excluding fragment".